pilot/pkg/proxy/envoy/v2/eds.go

```diff
@@ -125,14 +163,128 @@ func localityLbEndpointsFromInstances(instances []*model.ServiceInstance) []endp
 			}
 			localityEpMap[locality] = locLbEps
 		}
-		log.Infoa("EDS LOCALITY ep: ", (*lbEp).String(), localityEpMap)
 		locLbEps.LbEndpoints = append(locLbEps.LbEndpoints, *lbEp)
 	}
-	log.Infoa("EDS LOCALITY: ", localityEpMap)
 	out := make([]endpoint.LocalityLbEndpoints, 0, len(localityEpMap))
 	for _, locLbEps := range localityEpMap {
 		out = append(out, *locLbEps)
-		log.Infoa("EDS LOCALITY add: ", locLbEps.String())
 	}
 	return out
 }
+
+// StreamEndpoints implements xdsapi.EndpointDiscoveryServiceServer.StreamEndpoints().
+func (s *DiscoveryServer) StreamEndpoints(stream xdsapi.EndpointDiscoveryService_StreamEndpointsServer) error {
+	ticker := time.NewTicker(responseTickDuration)
```

Comment:
it would be good to have a comment why this function is behaving this way i.e. periodically sending endpoints. It feels like polling rather than streaming. At least just say it's a first pass implementation to avoid confusion.